src/connectors/configure/connectConfigure.js

```diff
@@ -45,7 +30,11 @@ Full documentation available at https://community.algolia.com/instantsearch.js/v
 export default function connectConfigure(renderFn = noop, unmountFn = noop) {
   return (widgetParams = {}) => {
     if (!isPlainObject(widgetParams.searchParameters)) {
-      throw new Error(usage);
+      throw new Error(
+        withUsage(
+          `The \`searchParameters\` option must be an object (got type "${typeof widgetParams.searchParameters}")`
```

Comment:
There is a gotcha
`.isPlainObject(null) === false` but `typeof null === object`.
In this case the error message will not make sense. May be best to remove the typeof part